js/src/main/scala/org/widok/bindings/Bootstrap/package.scala

```diff
@@ -141,6 +148,8 @@ package object Bootstrap {
     def element = DOM.createElement("button", contents)
   }
 
+  def SubmitButton(contents: View*) = Button(contents: _*).attribute("type", "submit")
```

Comment:
This is not related to Bootstrap. You could add a helper function to [``Button``](https://github.com/widok/widok/blob/master/js/src/main/scala/org/widok/bindings/HTML.scala#L81) to set the ``type`` more easily.